### PR TITLE
[PFCWD]: set default configuration when enabled by default

### PR DESCRIPTION
**Pull request by sihuihan88**
Signed-off-by: Sihui Han <sihan@microsoft.com>

**- What I did**
Set default configuration when enabled by default
**- How I did it**
Implement the pfcwd start_default function to set default configuration.
**- How to verify it**
Test on DUT
**- Previous command output (if the output of a command-line utility has changed)**

**- New command output (if the output of a command-line utility has changed)**

-->

### CODE REVIEW

File: pfcwd/main.py [hunk 1]
```diff
@@ -162,5 +169,34 @@ def stop(ports):
...
+
+    pfcwd_info = {
+        'detection_time': DEFAULT_DETECTION_TIME * max(port_num/DEFAULT_PORT_NUM, 1),
+        'restoration_time': DEFAULT_RESTORATION_TIME * max(port_num/DEFAULT_PORT_NUM, 1),
+        'action': DEFAULT_ACTION
+    }
+
+    for port in vlan_members:
+        configdb.mod_entry("PFC_WD_TABLE", port, None)
+        configdb.mod_entry("PFC_WD_TABLE", port, pfcwd_info)
```
Comment (lguohan) [inline pfcwd/main.py]: set_entry will remove extra fields. use set_entry instead of mod_entry.

File: pfcwd/main.py [hunk 2]
```diff
@@ -162,5 +169,34 @@ def stop(ports):
...
+       return
+    device_type = configdb.get_entry('DEVICE_METADATA', 'localhost').get('type')
+    if device_type != "ToRRouter":
+        return
+    port_num = len(configdb.get_table('PORT').keys())
+    vlan_members = [p[1] for p in configdb.get_table('VLAN_MEMBER').keys()]
+
+    pfcwd_info = {
+        'detection_time': DEFAULT_DETECTION_TIME * max(port_num/DEFAULT_PORT_NUM, 1),
+        'restoration_time': DEFAULT_RESTORATION_TIME * max(port_num/DEFAULT_PORT_NUM, 1),
```
Comment (lguohan) [inline pfcwd/main.py]: let's not do the calculation here, just use the DEFAULT_DETECTION_TIME. The reason is that the detection time is based on external network requirement, it should not be set based on the port num in the system.

File: pfcwd/main.py [hunk 3]
```diff
@@ -162,5 +169,34 @@ def stop(ports):
...
+        'action': DEFAULT_ACTION
+    }
+
+    for port in vlan_members:
```
Comment (lguohan) [inline pfcwd/main.py]: using vlan to determine whether the port is connected to server is the last resort, should use neighbor table first. check this commit.

https://github.com/Azure/sonic-buildimage/commit/720c71399c9b4d7d0e5a6083b3f63b887f3f6201

File: pfcwd/main.py [hunk 4]
```diff
@@ -5,6 +5,13 @@
 from tabulate import tabulate
 from natsort import natsorted
 
+# Default configuration
+DEFAULT_DETECTION_TIME = 200
+DEFAULT_RESTORATION_TIME = 200
+DEFAULT_POLL_INTERVAL = 200
+DEFAULT_PORT_NUM = 32
```
Comment (lguohan) [inline pfcwd/main.py]: we do not need this.

File: pfcwd/main.py (hunk 3 above)
Comment (sihuihan88) [reply pfcwd/main.py]: I checked the above commit and I believe it also uses vlan to determine the server facing port as we don't have neighbor table for servers.

**Pull request merged**